cogs/deadline.py

```diff
@@ -147,6 +156,10 @@ async def duedate_error(self, ctx, error):
                       help="delete a specific reminder using course name and homework name using "
                       "$deletereminder CLASSNAME HW_NAME ex. $deletereminder CSC510 HW2 ")
     async def deleteReminder(self, ctx, courseName: str, hwName: str):
+        if not ctx.channel.name == 'assignments':
+            await ctx.author.send('Please send questions to the #assignments channel.')
```

Comment:
'Please send questions to the #assignments channel.'

Should be:

'Please use this command inside the #assignments channel.'